tests/cypress/e2e/features/masks_basics.js

```diff
@@ -163,6 +163,7 @@ context('Manipulations with masks', { scrollBehavior: false }, () => {
     describe('Tests to make sure that empty masks cannot be created', () => {
         beforeEach(() => {
             cy.removeAnnotations();
+            cy.saveJob('PUT');
```

Comment:
Could you please clarify why it worked before?

Reply:
`cy.saveJob('PATCH', 200, 'removeUnderlyingPixelsUndoRedo')` lines inside `Empty masks are deleted using remove underlying pixels feature` case are depending on `PATCH` annotations requests. But after `removeAnnotations`, first update of the annotations is sent via `PUT` request. 

So before the fix first call of the `cy.saveJob('PATCH'...` catched not the `PATCH /annotations` , but `PATCH data/meta` and worked well. After the fix  `PATCH data/meta` is not sent, so the test failed.

We can eighter: 
1. Look for `PUT` request in first call of the `cy.saveJob(...` inside the test 
2. Save job after each remove annotations, so all saves inside test are looking for `PATCH` requests. 
Ive decided to go the second way.